CODE/logicytics/FileManagement.py

```
@@ -0,0 +1,234 @@
+            """
+            Zips files, generates a SHA256 hash, and moves the files.
+
+            Args:
+                path (str): The directory path containing the files.
+                name (str): The base name for the output files.
+                flag (str): A flag to be included in the output file names.
+
+            Returns:
+                tuple or str: A tuple containing success messages or an error message.
+            """
+            today = datetime.now()
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)